xendit_pos/static/src/js/payment_xendit_pos.js

```diff
@@ -65,7 +66,19 @@ odoo.define('xendit_pos.payment', function (require) {
             const self = this;
 
             const order = this.pos.get_order();
+            const payment_line = order.selected_paymentline;
+
+            if(payment_line.amount <= 0){
+                // TODO check if it's possible or not
```

Comment:
I think this might need to be reworded to explain what are we checking what's possible

Reply:
Updated @ikanyu